polyswarm/info.json

```diff
@@ -0,0 +1,230 @@
+{
+  "name": "polyswarm",
+  "label": "PolySwarm",
+  "version": "1.0.0",
+  "description": "PolySwarm is a real-time threat intelligence from a crowdsourced network of security experts and antivirus companies.",
+  "publisher": "Fortinet",
+  "cs_approved": true,
+  "cs_compatible": true,
+  "category": "Threat Intelligence",
+  "icon_small_name": "small.png",
+  "icon_large_name": "large.png",
+  "help_online": "",
+  "configuration": {
+    "fields": [
+      {
+        "name": "api_key",
+        "title": "API Key",
+        "type": "text",
+        "editable": true,
+        "visible": true,
+        "required": true,
+        "tooltip": "Specify the PolySwarm API Key",
+        "description": "Specify the PolySwarm API Key"
+      },
+      {
+        "title": "Verify SSL",
+        "description": "Specifies whether the SSL certificate for the server is to be verified or not. By default, this option is set as True.",
+        "required": false,
+        "editable": true,
+        "visible": true,
+        "type": "checkbox",
+        "name": "verify_ssl",
+        "value": true
+      }
+    ]
+  },
+  "operations": [
+    {
+      "operation": "url_reputation",
+      "title": "Scan URL",
+      "annotation": "url_reputation",
+      "description": "Uploads a URL PolySwarm and retrieves the analysis results.",
+      "category": "investigation",
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "url",
+          "title": "URL",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "The URL to scan.",
+          "description": "Queries PolySwarm for URL reputation information."
+        }
+      ],
+      "output_schema": {
+        "positives": "",
+        "total": "",
+        "permalink": ""
+      }
+    },
+     {
+      "operation": "ip_reputation",
+      "title": "Scan IP",
+      "annotation": "ip_reputation",
+      "description": "Uploads a IP to PolySwarm and retrieves the analysis results.",
+      "category": "investigation",
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "url",
+          "title": "IP",
+          "type": "text",
+          "editable": true,
+          "visible": true,
+          "required": true,
+          "tooltip": "The IP to scan.",
+          "description": "Queries PolySwarm for IP reputation information."
+        }
+      ],
+      "output_schema": {
+        "positives": "",
+        "total": "",
+        "permalink": ""
+      }
+    },
+     {
+      "operation": "domain_reputation",
+      "title": "Scan Domain",
+      "annotation": "domain_reputation",
+      "description": "Uploads a URL, IP, domain to PolySwarm and retrieves the analysis results.",
+      "category": "investigation",
+      "visible": true,
+      "enabled": true,
+      "parameters": [
+        {
+          "name": "url",
```

Comment:
the variable name can be changed to domain

Reply:
changed to artifact